content/docs/user-guide/project-structure/internal-files.md

```diff
@@ -120,3 +124,30 @@ $ cat .dvc/cache/19/6a322c107c2572335158503c64bfba.dir
 ```
 
 That's how DVC knows that the other two cached files belong in the directory.
+
+### Run-cache
+
+`dvc repro` and `dvc run` by default populate and reutilize a log of stages that
+have been run in the project. It is found in the `runs/` directory inside the
+cache (or [remote storage](/doc/command-reference/remote)).
```

Comment:
Other than in the DVC Internals guide should this be elsewhere?

If anyone can think of scenarios/examples/how-tos we should consider to mention the run-cache please lmk.